CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+# dbt_netsuite_source v0.10.0
+
+This is part of a batch release. 
+
+## 🎉 Feature Updates 🎉
+- Fields added to support account hierarchies in the `dbt_netsuite` end model. [PR #45](https://github.com/fivetran/dbt_netsuite_source/pull/45)
```

Comment:
Worth calling out that this is for Netsuite2
```suggestion
- Fields added to support account hierarchies in the `dbt_netsuite` Netsuite2 end model. [PR #45](https://github.com/fivetran/dbt_netsuite_source/pull/45)
```

Reply:
Updated.